odata-csdl/Appendix.md

```diff
@@ -152,6 +152,254 @@ https://www.ogc.org/standard/sfa/.
 $$$isec$$$
 :::
 
+-------
+
+# Appendix ##asec Model elements, properties and relationships
+
+## ##subasec Overview
+
+This appendix describes the model elements, their properties and relationships of the abstract OData model. The XSD schema is the normative document for the model elements and their properties. Yet, because of the limitations of XSD schema, it only contains the parent-child relationships of the model elements. For example the type of a property is represented as a string in XML and JSON, a string of the qualified name of the type. In the abstract model, it is a reference to the element that is the type.
+
+## ##subasec Model elements, properties and relationships
+
+The model is represented as a set of elements, each with a name, a set of properties and a set of child elements. This is treating the abstract OData model as a [labeled property graph](https://en.wikipedia.org/wiki/Property_graph). And the following is the schema of that labeled property graph.
```

Comment:
In the Wikipedia definition, the property values are only strings and numbers, but here we also have references and paths as property values.